CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+    - [Compiled Assets](#compiled-assets)
+  - [Release Process](#release-process)
+    - [Releases to WordPress.org](#releases-to-wordpressorg)
+
+## Code of Conduct
+
+Please read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our open source
+community approachable and respectable.
+
+## Contribution Guidelines
+
+Contributions to projects can be in the form of bug reports or feature requests
+via issues or pull requests. Pull requests are the preferred method of
+contribution to a project.
+
+### Reporting Issues/Bugs
+
+Issues are the best way to file a bug report for a project. Before filing an
+issue, please check for the following:
+
+- Attempt to replicate your problem to ensure it can be repeated and isn't
+  coincidental.
+- Ensure that your feature request is within the scope of the project. For
+  example, please don't suggest adding Drupal support to a WordPress plugin. If
+  you are unsure, try making a discussion post in the project to ask the project
+  maintainers.
+- Search the repository for an existing issue/pull request to your issue/feature
+  request. Check to make sure that your issue wasn't already reported/requested.
+
+## Maintainership
+
+All repositories should have a [CODEOWNERS](codeowners) file included with a
+specified owner for the project. Preferably a team or 2+ individuals would be
+marked as "owners" who are responsible for the project. These owners are
+responsible for the maintainership of the project and controlling the scope of
+features within the said project. They shall have the final say on whether or not a
+feature request is a good fit for the scope of a project.
+
+## Best Practices
+
+The following are best practices that we strive for in all of our projects. A
+project may override these where it makes sense. The respective project should
+have an updated `CONTRIBUTING.md` included with information about best practices
+for the project.
+
+### Branch Names
+
+The default branch for open-source repositories is `develop`. Always work from
+the `develop` branch and open up pull requests against it.
+
+The default branch name of `develop` only applies to 'package' repositories that
+have the intended purpose of being distributed in a standalone
+plugin/library/framework/etc. Hosted packages, those where the repository
+represents a hosted website, should use `main` as a default branch name.
+
+Projects should always reference a tagged version of a project and never track
+`develop`/`dev-develop`. The `develop` branch is considered 'nightly' and
+unstable at all times and is subject to change without notice. Releases that
+follow the [release process](#release-process) should always be backward
+compatible and well documented via a `CHANGELOG`.
+
+### Coding Style
+
+If the project maintainer has any additional requirements, you will find them
+listed here.
+
+#### Alley Coding Standard
+
+Code must generally follow the [Alley Coding Standard](alley-coding-standards)
```

Comment:
This link and similar ones don't appear to work.  Is that just a preview issue, or do these need to be complete URLs?

Reply:
If you preview the markdown doc it will render the link from below. Believe it is a footnote?

https://github.com/alleyinteractive/.github/pull/7/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R154